blogsite/blogsite/views.py

```diff
@@ -0,0 +1,52 @@
+from django.http import HttpRequest, HttpResponse
+    
+def home_view(request):
+    return HttpResponse("Wellcome")
+
+def about_view(request):
+    return HttpResponse("About website")
+
+def create_article(request):
+    return HttpResponse("Create article")
+
+def coment_article(request, article):
+    return HttpResponse(f"Coment to article -{article}")
+
+def update_article(request, article):
+    return HttpResponse(f"Update to article -{article}")
+
+def delete_article(request, article):
+    return HttpResponse(f"Delete to article - {article}")
+
+def topics(request):
+    return HttpResponse("Page with topics")
+
+def topics_subscribe(request, topic):
+    return HttpResponse(f"Sybscripe on topic - {topics}")
```

Comment:
Тут не вірну зміну вказав, але то не критично.
Все одне це буде мінятись у наступних завданнях.

```suggestion
    return HttpResponse(f"Sybscripe on topic - {topic}")
```